nestkernel/recording_device.h

```diff
@@ -42,6 +42,62 @@
 namespace nest
 {
 
+/* BeginUserDocs: NOINDEX
+
+Recording time window
++++++++++++++++++++++
+
+The time span during which the recorder actually records, can be
+specified using the properties ``start`` and ``stop``. These define
```

Comment:
```suggestion
The time span during which the recorder actually records can be
specified using the properties ``start`` and ``stop``. These define
```

Reply:
Can we perhaps replace the word "actually" with "actively"?

Reply:
Yes.
```suggestion
The time span during which the recorder actively records, can be
specified using the properties ``start`` and ``stop``. These define
```